src/main/kotlin/org/malachite/estella/services/MailService.kt

```diff
@@ -3,18 +3,23 @@ package org.malachite.estella.services
 import org.malachite.estella.commons.models.interviews.Interview
 import org.malachite.estella.commons.models.offers.Application
 import org.malachite.estella.commons.models.offers.Offer
+import org.malachite.estella.commons.models.people.Organization
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.http.HttpEntity
+import org.springframework.stereotype.Component
 import org.springframework.web.client.RestTemplate
 
-object MailService {
+@Component
+class MailService(
+    @Value("\${mail_service_url}") val SERVICE_URL: String
```

Comment:
To pobiera dane z propertiesów?

Reply:
yup

Reply:
Jakbyś chciał zagnieżdżone, to podajesz w stringu po kropkach